TrafficCapture/trafficReplayer/README.md

```diff
@@ -39,9 +39,9 @@ over the constructed JSON document.
 Since payloads can be arbitrarily complex (compression, chunking), and may not be subject to any changes via the
 transformation rules, the HttpJsonTransformer creates the channel pipeline *only* to parse the HTTP headers. The
 transformation is run on this partial, in-construction
-[HttpJsonMessageWithFaultablePayload](./src/main/java/org/opensearch/migrations/replay/datahandlers/http/HttpJsonMessageWithFaultingPayload.java)
+[HttpJsonMessageWithFaultablePayload](../../coreUtilities/src/main/java/org/opensearch/migrations/transform/HttpJsonMessageWithFaultingPayload.java)
```

Comment:
This file is super specific to the replayer.  Even where other components will need to do transformations, they probably won't need to optimize away reading payload streams.  We do that in the replayer because it might mean that we can leave bytebuffs go right through the netty pipeline directly and never need to aggregate or transform them.  That adds up for the 10MB+ requests that aren't uncommon for opensearch.

For use cases outside of the replayer, I'd suspect that somebody already has the components of an IHttpMessage and just writes a quick bridge.

Other files that I wouldn't move out.
* JsonKeysForHttpMessage
* HttpJsonMessageWithFaultingPayload.java
* PayloadNotLoadedException.java
* PayloadAccessFaultingMap
* RemovingAuthTransformerFactory
* ListKeyAdapting...Headers
* StrictCaseInsensitive...Headers

I don't think that the remaining transforms classes belong in coreUtilities.  I also don't think that you'll want to make the awsUtilities package take a dependency on coreUtilities.  Instead of relying upon an IHttpMessage, you could inline IHttpMessage to be verb, uri, protocol, & headers.  Then you don't need to worry about having too many adapter shims laying around and the need to couple libraries together.